Formula/apache-brooklyn-cli.rb

```diff
@@ -0,0 +1,62 @@
+require "language/go"
+require "socket"
+require "timeout"
```

Comment:
`socket` and `timeout` are only needed in the test, thus let's move these `require`s to the `test do` block. We want to avoid `require`ing them every time the formula is loaded (e.g. during `brew info`).